addons/purchase_stock/views/purchase_views.xml

```diff
@@ -49,7 +49,7 @@
             <xpath expr="//field[@name='order_line']/form//field[@name='invoice_lines']" position="after">
                 <field name="move_ids"/>
             </xpath>
-            <xpath expr="//field[@name='order_line']/form//field[@name='account_analytic_id']" position="before">
+            <xpath expr="//field[@name='order_line']/tree//field[@name='account_analytic_id']" position="before">
```

Comment:
I don't see a reason to remove it from the form view.
I'd rather have another `xpath` for the tree view instead, on which you'd add `optional="hide"` as well, as we clearly don't want to see that column by default when the debug mode is activated.

Reply:
Ok, thanks for the suggestions, PR updated